doc/Yang Model Guidelines.md

```diff
@@ -71,10 +74,10 @@ Leaf names are same PACKET_ACTION, IP_TYPE and PRIORITY.
 ```
 
 
-### 3. Hierarchy of the an objects in yang models will be same as for all the objects at same hierarchy in Redis. If any exception is created then it must be recorded properly.
-For Example: 
+### 3. Data Node/Object Hierarchy of the an objects in YANG models will be same as for all the objects at same hierarchy in Redis. If any exception is created then it must be recorded properly.
```

Comment:
The step does not tell how to record exception. Will the exception be understood by processing library (e.g. CVL). Please provide example of exception and how to record the same.